src/reuse/helper/errorHandler.ts

```diff
@@ -66,4 +72,49 @@ export default class ErrorHandler implements IErrorHandler {
       return "";
     }
   }
+
+  private _retrieveLastLevelFunctionNameFromStack(errorObject: unknown | Error): string {
+    if (errorObject instanceof Error && errorObject.stack) {
+      const regex = /\bat\s*(.+?)\(/;
+      let functionName: string = "block";
+      let initFunctionArray: string[] = [];
+
+      let errorStackAfterSplit = errorObject.stack.split("\n");
+
+      for (let i = 0, index = 0; i < errorStackAfterSplit.length; i++) {
+        let matchedString = !this._isThirdPartyModuleExistInStack(errorStackAfterSplit[i]) && errorStackAfterSplit[i].match(regex);
+
+        if (matchedString) {
+          initFunctionArray[index] = matchedString[1].trim();
+
+          if (initFunctionArray[index].includes("anonymous")) {
+            index = index - 1;
+            functionName =
+              index < 0
+                ? functionName
+                : initFunctionArray[index].includes(".")
+                ? initFunctionArray[index].substring(initFunctionArray[index].indexOf(".") + 1)
+                : initFunctionArray[index];
+            break;
+          }
+          index++;
+        }
+      }
+      return functionName;
+    } else {
+      return "";
+    }
+  }
+
+  private _isThirdPartyModuleExistInStack(line: string): boolean {
```

Comment:
does exists/ is included

```suggestion
  private _isThirdPartyModuleIncludedInStack(line: string): boolean {
```